internal/generator/vector/normalize.go

```diff
@@ -95,6 +95,13 @@ func NormalizeLogs(spec *logging.ClusterLogForwarderSpec, op generator.Options)
 		el = append(el, NormalizeOpenshiftAuditLogs(RawOpenshiftAuditLogs, OpenshiftAuditLogs)...)
 		el = append(el, NormalizeOVNAuditLogs(RawOvnAuditLogs, OvnAuditLogs)...)
 	}
+
+	for _, input := range spec.Inputs {
+		if input.Source != nil && input.Source.HTTP != nil && input.Source.HTTP.LogType == `audit` {
+			el = append(el, NormalizeK8sAuditLogs(input.Source.HTTP.Name+`_items`, input.Source.HTTP.Name+`_normalized`)...)
```

Comment:
Looks like we need a helper function to ensure "normalized" is consistent between the 2 files